spec/index.bs

```diff
@@ -5858,6 +5858,7 @@ operation, not the semantics of the destination texture.
 dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
     GPUPredefinedColorSpace colorSpace = "srgb";
     boolean premultipliedAlpha = false;
+    boolean originBottomLeft = false;
```

Comment:
I would move `originBottomLeft` to `GPUImageCopyExternalImage` instead.

I think it makes more sense to describe which corner of the source image to start reading from, rather than which corner of the destination texture to start writing to. Technically it doesn't make any difference, but in the mental model I think textures should always be written in the same direction.